cypress/integration/premium/admin.spec.ts

```diff
@@ -343,132 +383,17 @@ describe("Premium tier - Global Admin user", () => {
   });
   describe("Manage software page", () => {
     beforeEach(() => {
-      cy.intercept("GET", "/api/latest/fleet/config", getConfig).as(
-        "getIntegrations"
-      );
       cy.visit("/software/manage");
-      cy.wait("@getIntegrations").then((configStub) => {
-        console.log(JSON.stringify(configStub));
-      });
     });
     // it(`displays "Probability of exploit" column`, () => {
     //   cy.getAttached("thead").within(() => {
     //     cy.findByText(/vulnerabilities/i).should("not.exist");
     //     cy.findByText(/probability of exploit/i).should("exist");
     //   });
     // });
-    it("allows admin to create webhook software vulnerability automation", () => {
```

Comment:
Moved to software.spec.ts instead of duplicated in both free/admin.spec.ts and premium/admin.spec.ts